docs/Deploying-Pega-on-AKS.md

```diff
@@ -360,12 +360,13 @@ To customize these files, you must download them from the source github reposito
 
 ```bash
   $ helm search repo pega
-  NAME        CHART VERSION   APP VERSION     DESCRIPTION
-  pega/pega   1.2.0                           Pega installation on kubernetes
-  pega/addons 1.2.0           1.0             A Helm chart for Kubernetes
+  NAME              CHART VERSION   APP VERSION     DESCRIPTION
+  pega/pega             1.2.0                           Pega installation on kubernetes
+  pega/addons           1.2.0           1.0             A Helm chart for Kubernetes
+  pega/backingservices  1.2.0                           Helm Chart to provisioning backing services for Pega (eg. Search and Reporting Service)
 ```
 
-These two charts in this /charts/pega folder of the pega-helm-charts repository, pega and addons, require customization for your organization's AKS deployment of Pega Platform.
+The two charts in this /charts/pega folder of the pega-helm-charts repository, pega and addons, require customization for your organization's AKS deployment of Pega Platform. The third chart `backingservices` is optional but recommended for Pega platform versions 8.6 and above.
```

Comment:
The  `pega` and `addons` charts in the /charts/pega folder require customization for your organization's AKS deployment of Pega Platform. The `backingservices` chart is optional, but recommended for Pega Infinity 8.6 and later.